CellView.pde

```diff
@@ -105,18 +105,20 @@ class CellView extends ViewBase {
                     vertex(50 - 0.15 * energySymbolSize, 50 - 0.15 * energySymbolSize);
                     endShape(CLOSE);
 
-                    if(screenSize > 75) {
+                    if(screenSize > 80) {
                         float progressToNextCodonTick = norm(cellModel.ticksSinceLastCodonTick, 0, cellModel.ticksPerCodonTick);
+                        ArrayList<CodonBaseModel> codonModels = cellModel.codonModels;
```

Comment:
Do you mean to make a copy of codonModels. If so, you should write:

ArrayList<CodonBaseModel> codonModels = (ArrayList<CodonBaseModel>)cellModel.codonModels.clone();